src/java/com/twitter/common/styleguide.md

```diff
@@ -13,7 +13,38 @@ In general, much of our style and conventions mirror the
 [Code Conventions for the Java Programming Language](http://www.oracle.com/technetwork/java/codeconvtoc-136057.html)
 and [Google's Java Style Guide](https://google.github.io/styleguide/javaguide.html).
 
-[TOC]
+
+## Table of contents
+
+- [Recommended reading](https://github.com/knjk04/commons/blob/master/src/java/com/twitter/common/styleguide.md#recommended-reading)
```

Comment:
Sweet! How’d you generate this? Mind replacing the `knjk04` with `twitter`? 🙏

Reply:
I typed this out by hand! Yeah, sure. Sorry, I didn't realise I was getting the links from my for :)

I can look into ways to generate the TOC. It's probably easier for everyone else.

Reply:
Update: I've made the changes you asked for in a different branch. See PR: https://github.com/twitter/commons/pull/467